tests/fakeprotocol_iot.py

```diff
@@ -189,40 +189,47 @@ async def query(self, request, retry_count: int = 3):
 
 
 class FakeIotTransport(BaseTransport):
-    def __init__(self, info, fixture_name=None):
+    def __init__(self, info, fixture_name=None, *, verbatim=False):
         super().__init__(config=DeviceConfig("127.0.0.123"))
         info = copy.deepcopy(info)
         self.discovery_data = info
         self.fixture_name = fixture_name
         self.writer = None
         self.reader = None
-        proto = copy.deepcopy(FakeIotTransport.baseproto)
+        self.verbatim = verbatim
+
+        proto = {} if verbatim else copy.deepcopy(FakeIotTransport.baseproto)
 
         for target in info:
-            # print("target %s" % target)
+            if self.verbatim:
+                proto[target] = info[target]
+                continue
```

Comment:
Deserves definitely a comment and/or restructuring to make it clear why this is being done.

Reply:
Done